.vscode/settings.json

```diff
@@ -1,15 +1,11 @@
 {
-  "cSpell.words": [
```

Comment:
Was going on here?

Reply:
Lol. I commented it out because I didn't like the green title bar. I accidentally pushed it. I'll revert it back now.

Reply:
I've reverted it @IyiKuyoro